x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -615,25 +684,48 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
           </EuiContextMenuItem>
         );
       }
+    }
 
-      if (anomaly.isGeoRecord === true) {
-        items.push(
-          <EuiContextMenuItem
-            key="view_in_maps"
-            icon="gisApp"
-            onClick={async () => {
-              const mapsLink = await getMapsLink(anomaly);
-              await application.navigateToApp(MAPS_APP_ID, { path: mapsLink?.path });
-            }}
-            data-test-subj="mlAnomaliesListRowActionViewInMapsButton"
-          >
-            <FormattedMessage
-              id="xpack.ml.anomaliesTable.linksMenu.viewInMapsLabel"
-              defaultMessage="View in Maps"
-            />
-          </EuiContextMenuItem>
-        );
-      }
+    if (anomaly.isGeoRecord === true) {
```

Comment:
Can we check if Maps is enabled for the user's role in deciding whether to show this, and the existing `View in Maps` items? If I log in as `ml_viewer`, who doesn't have Maps enabled, I get the following page when using the link:

<img width="592" alt="image" src="https://user-images.githubusercontent.com/7405507/187488730-00f6c9cd-b613-43f3-bd4a-d2adb0a208f9.png">

Reply:
Created issue for fix https://github.com/elastic/kibana/issues/139750